edgexfoundry-holding/device-onvif-camera
# build: Upgrade to Go 1.18, Alpine 3.16, linter version and latest SDK/go-mod versions

Also optimized attribution script

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A for Build PR**
- [ ] I have opened a PR for the related docs change (if not, why?) **N/A for Build PR**
  <link to docs PR>

## Testing Instructions
With Go 1.18 installed, run the following make commands:
- make build
- make test
- make docker

Verify commands have no errors.

## New Dependency Instructions (If applicable)
